libcudacxx/include/cuda/std/detail/libcxx/include/bit

```diff
@@ -336,30 +254,24 @@ inline _LIBCUDACXX_INLINE_VISIBILITY constexpr int __libcpp_ctz(unsigned __x)
   return __binary_ctz32(static_cast<uint64_t>(__x), 0);
```

Comment:
I was always wondering why w cast to 64bit for `__binary_ctz32`

Reply:
*We could* cast down to 32, 16, and 8. The GPU might thank us for this. But ideally this is only used in C++11.